docs/dm/assets.md

```diff
@@ -0,0 +1,170 @@
+This can be useful for hiding future encounters or placing enemies in stealth. 
+
+### Is locked
+
+Selecting this option locks the object in place.
+This can be useful for maps you don’t want to accidently move while playing.
+
+### Show badge
+
+Selecting this option displays a badge at the bottom right of the object.
+
+### Blocks vision/light
+
+Selecting this option blocks vision and light from passing through the object.
+More information about lighting can be found in the [lighting and vision](/docs/dm/light-shadows.md) document.
+
+### Blocks movement
+
+Selecting this option prevents other objects from moving through it.
+This is useful for creating walls in your dungeon you don’t want your players to pass through.
+
+### Border colour & Fill colour
+
+These options control the color of the borders around and the fill color of default tokens. 
+
+### Access
+
+Here is listed the players who have access to the asset.
+Selecting a player name and “add access” will allow them to interact with the asset.
+Selecting any of the icons next to “default” gives all players access to the asset.
+The pen icon allows those with access to edit the asset.
+The four arrows icon allows those with access to move the asset unless locked.
+The lightbulb icon allows those with access to see the asset. 
+
+### Trackers
+
+Trackers can track numerical values of assets, such as health to player tokens.
+To use, under “name” label the value you wish to track, such as health.
+Next, write the current value which the player has and the maximum value.
+By clicking the eye icon, you can toggle whether this information is public to all players or strictly visible to the DM.
+
+With these values established, when you click the asset, you will now see the values in a quick menu.
+You can quickly edit these values by clicking the number and typing in the new value.
+
+This can be used for health, gold, death saves, or any other value that you need to track. 
+
+### Auras
+
+Auras are visible light which emanates from an asset.
+These can be any color or strength you wish.
+
+More information about lighting can be found in the [lighting and vision](/docs/dm/light-shadows.md) document.
+
+### Labels
+
+![](/docs/dm/label-manager.png)
+
+Labels allow the DM to categorize objects.
+To start, type a category at the bottom left.
+Then, type a name within that category.
+This will appear above.
+Under Cat. is the category, and under Name is the specific member of that category.
+
+To assign an asset its category, click on the label in the list.
+To the right is an eye icon, which makes this label public.
+Beside that is the option to delete the label.
+
+Once these categories have been defined, you can click “filter” in the bottom right corner.
+This will present a menu of all categories.
+Enabling one shows you all assets with that category.
+
+### Annotation
+
+At the bottom of the Edit asset menu is a space to write anything localized on that one asset.
```

Comment:
These annotations will also appear at the top of your screen when you mouseover the shape!